documentation/dotnet-monitor.md

```diff
@@ -0,0 +1,309 @@
+
+The duration of collection can be customized via the `durationSeconds` querystring parameter The diagnostic data can be customized via the `profile` querystring parameter to include any combination of the preset profiles:
+
+- `CPU` (CPU profiler),
+- `Http` (Request start/stop events from ASP.NET Core),
+- `Logs` (Logging from the `EventSourceLogger` from `Microsoft.Extensions.Logging` library); and
+- `Metrics`(Runtime and ASP.NET Core `EventCounters`).
+
+For example, a request to  `/trace?profile=cpu,logs` will enable the collection of just the CPU profiler and logs.
+
+In addition to the `GET` endpoint, there is `POST` version of the endpoint that allows you to specify what `EventSource` providers to enable via the request body. 
+
+To collect a trace of the target process, run the following command:
+
+In PowerShell,
+
+```powershell
+PS> Start-Process http://localhost:52323/trace
+```
+
+In bash,
+```bash
+$ wget --content-disposition http://localhost:52323/trace
+```
+
+The resulting `.nettrace` file can be analyzed with both Visual Studio and PerfView.
+
+### Logs
+
+The `/logs` endpoint will stream logs from the target process for a duration of 30 seconds.
+
+The duration of collection can be customized via the `durationSeconds` querystring parameter. The logs endpoint is capable of returning either newline delimited JSON([application/x-ndjson](https://github.com/ndjson/ndjson-spec)) or the Event stream format([text/event-stream](https://developer.mozilla.org/docs/Web/API/Server-sent_events/Using_server-sent_events#Event_stream_format)) based on the specified `Accept` header in the HTTP request.
+
+To start streaming log from the target process, run the following command:
+
+In PowerShell,
+
+```powershell
+PS> Start-Process http://localhost:52323/logs
+```
+
+In bash,
+```bash
+$ curl -H "Accept:application/x-ndjson" http://localhost:52323/logs --no-buffer
+```
+
+### Metrics
+
+The `/metrics` endpoint will return a snapshot of runtime and ASP.NET Core metrics in the [prometheus exposition format](https://prometheus.io/docs/instrumenting/exposition_formats/#text-based-format). Unlike the other diagnostics endpoints, the metrics endpoint will not be available if `dotnet-trace` detects more than one target process. In addition to being accessible via the URLs configured via the `--urls` parameters, the metrics endpoint is also accessible from the URLs configured via the `--metricUrls`. When running in Kubernetes, it may be suitable to expose the metrics URL to other services in your cluster to allow them to scrape metrics.
+
+While metrics collection is enabled by default when `dotnet-monitor` detects exactly one target process, it can configured to disable to collection of metrics entirely via the `--metrics` parameter. In the example below, metrics collection will not be enabled.
```

Comment:
I'd expect people will prioritize learning how it works before learning how to turn it off

Reply:
Makes sense. Will move to the end of the section